app/lib/submission_builder/document.rb

```diff
@@ -18,6 +18,8 @@ def document
 
     def determine_default_schema_version_by_tax_year
       case @submission.tax_return&.year || @submission.data_source&.tax_return_year
+      when 2024 # TODO: update with 2024 schema when available
+        "2023v5.0"
```

Comment:
not sure where we can get this

Reply:
i tried to look for all usages of `@schema_version` in children of this class and as far as i can tell it's only used for federal stuff? so we should be good

Reply:
(federal stuff = building federal returns i.e. CTC code)